dw-menu.js

```diff
@@ -105,10 +145,24 @@ export class DwMenu extends DwCompositeDialog {
 
   connectedCallback() {
     this._setDialogConfig();
+    this._showHeader = Boolean(this.heading) || this.showClose;
```

Comment:
@hitendreamworld computed property value should be set from `willUpdate` in general practice. So, please update this implementation

Reply:
DwCompositeDialog uses the old `lit-element` version and that library does not provide `willUpdate` lifecycle. So, I wrote a custom getter/setter for `heading` and `showClose`.